cpmonitor/views.py

```diff
@@ -122,7 +130,7 @@ def cap_checklist(request, city_slug):
     return render(request, "cap_checklist.html", context)
 
 
-def get_cap_checklist(city):
+def get_cap_checklist(city) -> dict:
```

Comment:
I'd prefix the function with an underscore as python's way of denoting private functions as get_cap_checklist is only a helper to be used by view functions. I did the same for _calculate_summary. Also, this makes it easier to distinguish helpers from actual view functions.

Reply:
done